RecoTauTag/HLTProducers/interface/L1TJetsMatching.h

```diff
@@ -100,13 +99,14 @@ std::pair<std::vector<T>, std::vector<T>> L1TJetsMatching<T>::categorise(
 
     const T& myJet1 = (pfMatchedJets)[i1];
     const T& myJet2 = (pfMatchedJets)[i2];
+    const double M2jj = (Mjj > 0. ? Mjj * Mjj : -1.);
```

Comment:
```suggestion
    const double M2jj = (Mjj >= 0. ? Mjj * Mjj : -1.);
```
(preserves current behaviour for case `Mjj == 0`)

Reply:
Well, no: current behaviour is preserved with my expression, that's why I wrote it as such.
However, I agree that it can be improved/simplified following your suggestion, and I can implement it.

Reply:
> Well, no: current behaviour is preserved with my expression, that's why I wrote it as such.
> However, I agree that it can be improved/simplified following your suggestion, and I can implement it.

Uhm, no: even here you are right: your expression is the one that preserves the previous behaviour, and mine was chosen instead to simplify the (unlikely) case of a null invariant mass. Let stick on the original expression, as you are suggesting

Reply:
Sorry, I cross-posted. I think we agree.